content/en/Platform Deep Dive/Scans/reduced-scope.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Partial Scans: Reduced Scope"
+linkTitle: "Partial Scans: Reduced Scope"
+weight: 50
+description: Define a specific set of URLs within your target to scan
+---
+
+Rather than scanning the entire target, specify a subset of URLs within your target to scan by defining a Reduced Scope in your target settings. This is particularly useful when you want to focus on specific areas of your application, such as new features or high-risk areas.
+
+### Benefits
+
+- **Faster Feedback:** Get faster feedback on the security of your code changes. This is especially helpful in CI/CD pipelines, where you need to deliver code changes quickly and reliably.
```

Comment:
Does the public API support this already? If not, we might not want to mention CI/CD yet.

Reply:
Yes, they can use this endpoint to trigger a new scan https://cobalt-public-api.netlify.app/v2/#create-a-dast-scan